tools/gyp/buildbot/buildbot_run.py

```diff
@@ -73,41 +61,7 @@ def PrepareCmake():
   CallSubProcess( ['make', 'cmake'], cwd=CMAKE_DIR)
 
 
-def PrepareAndroidTree():
-  """Prepare an Android tree to run 'android' format tests."""
-  if os.environ['BUILDBOT_CLOBBER'] == '1':
-    print '@@@BUILD_STEP Clobber Android checkout@@@'
-    shutil.rmtree(ANDROID_DIR)
-
-  # The release of Android we use is static, so there's no need to do anything
-  # if the directory already exists.
-  if os.path.isdir(ANDROID_DIR):
-    return
-
-  print '@@@BUILD_STEP Initialize Android checkout@@@'
-  os.mkdir(ANDROID_DIR)
-  CallSubProcess(['git', 'config', '--global', 'user.name', 'trybot'])
-  CallSubProcess(['git', 'config', '--global',
-                  'user.email', 'chrome-bot@google.com'])
-  CallSubProcess(['git', 'config', '--global', 'color.ui', 'false'])
-  CallSubProcess(
-      ['repo', 'init',
-       '-u', 'https://android.googlesource.com/platform/manifest',
-       '-b', 'android-4.2.1_r1',
-       '-g', 'all,-notdefault,-device,-darwin,-mips,-x86'],
-      cwd=ANDROID_DIR)
-
-  print '@@@BUILD_STEP Sync Android@@@'
-  CallSubProcess(['repo', 'sync', '-j4'], cwd=ANDROID_DIR)
-
-  print '@@@BUILD_STEP Build Android@@@'
-  CallSubProcess(
-      ['/bin/bash',
-       '-c', 'source build/envsetup.sh && lunch full-eng && make -j4'],
-      cwd=ANDROID_DIR)
-
-
-def GypTestFormat(title, format=None, msvs_version=None):
+def GypTestFormat(title, format=None, msvs_version=None, tests=[]):
```

Comment:
Using `tests=[]` is practically not advisable. Use `tests=None` and then inside the function do,

```py
if tests is None:
    tests = []
```